corehq/warehouse/models/meta.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+
+from django_fsm import FSMField, transition
+
+from ..states import (
+    FACT_TABLE_NEEDS_UPDATING,
+    FACT_TABLE_BATCH_DUMPED,
+    FACT_TABLE_READY,
+    FACT_TABLE_FAILED,
+    FACT_TABLE_DECOMMISSIONED,
+)
+
+
+class FactTableState(models.Model):
```

Comment:
i think this table is very much in flux, so hard right to now to determine what exactly we'll need. it'll be easy to add and remove states as well